api/v1alpha1/shared_types.go

```diff
@@ -62,7 +62,11 @@ const (
 
 	SecretTypeKuadrantCoreDNS corev1.SecretType = "kuadrant.io/coredns"
 
-	DefaultProviderSecretLabel = "kuadrant.io/default-provider"
+	DefaultProviderSecretLabel                   = "kuadrant.io/default-provider"
+	EndpointGVRKey                               = "kuadrant.io/endpoint_gvr"
+	EndpointLabelSelectorKey                     = "kuadrant.io/zone_record_label"
+	EndpointLabelValueKey                        = "kuadrant.io/zone_record_label_value"
```

Comment:
```suggestion
	EndpointGVRKey           = "ENDPOINT_GVR"
	EndpointLabelSelectorKey = "ENDPOINT_ZONE_RECORD_LABEL"
```

Consistent with how all other providers work, these are not expected to be labels, but keys in the provider secret.